src/view/henvendelseList.less

```diff
@@ -1,10 +1,11 @@
 @import (reference) "~nav-frontend-core/less/core";
 
 
-.henvendelser-list {
+.henvendelse-list {
+  flex-grow: 1;
   overflow-y: scroll;
   padding: 1.5rem 1rem 1rem 1rem;
+  border-style: solid;
+  border-width: 0 0 0.2rem 0;
```

Comment:
Gjentar det @nutgaard sa jeg
Foretrekker border-bottom: 3px solid @navLysGra i disse tilfellene.
Imho er dette ett av de veldig få tilfellene der man kan bruke px istedet for rem, og det er lettere å se hva formålet bak stylingen er når man samler det på en linje sånn.